app/views/loans/viewloanaccountdetails.html

```diff
@@ -571,10 +572,14 @@
                                         {{transaction.outstandingLoanBalance | number}}
                                     </td>
 
-                                    <td> <a ng-if="transaction.manuallyReversed!=true" ng-click="viewloantransactionreceipts(transaction.id)">
+                                    <td> <a tooltip="{{'label.heading.viewreceipts' | translate }}" ng-if="transaction.manuallyReversed!=true" ng-click="viewloantransactionreceipts(transaction.id)">
                                         <i class="icon-file-text icon-large"></i>
                                     </a>
                                     </td>
+									 <td> <a tooltip="{{'label.anchor.viewjournalentries' | translate}}" ng-click="viewloantransactionjournalentries(transaction.id)">
+                                        <i class="icon-collapse-alt icon-large coll"></i>
```

Comment:
as discussed earlier, using a collapse icon here is misleading, can you please update the same?